bofire/strategies/strategy.py

```diff
@@ -26,6 +26,15 @@ def __init__(
         self._experiments = None
         self._candidates = None
 
+    def _get_seed(self) -> int:
+        """Returns an integer sampled from the strategies random number generator,
+        that can be used to seed dependent generators.
+
+        Returns:
+            int: random seed.
+        """
+        return int(self.rng.integers(1, 100000))
```

Comment:
Out of interest: This is just a very large number?

Reply:
can you use `sys.maxsize`? https://stackoverflow.com/questions/7604966/maximum-and-minimum-values-for-ints

Reply:
Yep, it is just a large number, botorch is using the same I think, for this reason, I would stay with this one.